src/torrent.cpp

```diff
@@ -8399,15 +8399,14 @@ namespace libtorrent {
 	}
 	catch (...) { handle_exception(); }
 
-	void torrent::on_torrent_aborted() try
+	void torrent::on_torrent_aborted()
```

Comment:
does it makes sense to remove the `try` above in `on_cache_flushed` too?

Reply:
``on_cache_flushed`` is still doing things that could throw an exception though